### DISCUSSION TITLE
Turn comments into replies

### DISCUSSION THREAD

**Discussion by fonsp** [community/community#3473] [Discussions]
I LOVE the fact that Discussions have two layers: Comments to respond to the original question, and Replies to individual Comments. This is exactly what I was missing in GitHub Issues.

### Proposal
**It would be great if moderators could turn Comments into Replies.** This would be especially useful when converting a GH Issue to a GH Discussion: right now, an Issue like:

```
#> Hey! My issue is...

> Did you try ...?

> Yes, I did! But ...

> You can also try ...
```

becomes a Discussion with three comments. But ideally, a moderator could turn this into a Discussion with the correct hierachy:

```
#> Hey! My issue is...

> Did you try ...?
>> Yes, I did! But ...

> You can also try ...
```

**Comment by dgw** [comment]
Enthusiastically second this request!

I've run into a user who's oblivious to the "Reply" feature and keeps responding to things as top-level comments. It makes the discussion hard to follow, and it would be great if one of us repo admins could merge the several comment threads together as they were intended.

A follow-up idea: If a discussion already has comments in response to it, perhaps the large "Write" field at the bottom could be collapsed? I think the big text field is why this user (who's previously demonstrated minimal understanding of both GitHub's and $our_project's features) keeps posting top-level comments without noticing the option to reply in threads.

**Reply by michael-hawker** [reply]
Yeah, I've been noticing this behavior a ton too lately. It's very confusing for folks. Something definitely needs to be adjusted, but it'd be really nice to have move moderation tools to move a response into a reply and vice-versa.
